src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3726,6 +3735,7 @@ impl<T: AstInfo> AstDisplay for WithOptionValue<T> {
                 WithOptionValue::DataType(_)
                 | WithOptionValue::Item(_)
                 | WithOptionValue::UnresolvedItemName(_)
+                | WithOptionValue::Ident(_)
```

Comment:
I thought that object names did need to be redacted? Though I can't quite remember and don't know where the rules are written down.

Reply:
Hmm I'm not sure - but I am following the convention of the `Item` and `UnresolvedItemName` types here which themselves are identifiers too.